app/containers/Vaults/index.js

```diff
@@ -468,7 +468,21 @@ const VaultsWrapper = (props) => {
   const migratedVaultsWithBalance = [];
   if (isMigrated) {
     vaultItems.forEach((vault) => {
-      if (vault && vault.balanceOf && vault.balanceOf[0].value > 0) {
+      let decimals = 18;
+      if (vault && vault.token && vault.token.decimals > 0) {
+        if (vault.token.decimals > 6) {
+          decimals = vault.token.decimals - 4;
+        } else if (vault.tokens.decimals >= 4) {
```

Comment:
> } else if (vault.tokens.decimals >= 4) {

Typo here, should be
`vault.token.decimals`

Reply:
pushed a fix thanks